apps/web/components/Modal.tsx

```diff
@@ -61,25 +61,24 @@ export function Modal(props: ModalProps) {
               leave="ease-in duration-200"
               leaveFrom="opacity-100 scale-100"
               leaveTo="opacity-0 scale-95"
+              className={clsx(
```

Comment:
modals everywhere else on the site still work fine after this change?

Reply:
I've tested the modals, only VideoDemo.tsx (Landing page demo video) had a layout issue which will be fixed in the next commit!

Reply:
Yeah, the only place where layout was not looking good was the VideoDemo.tsx modal (Demo video on the landing page) which is now fixed in the latest PR linked with issue #221